src/composables/forms.ts

```diff
@@ -1,6 +1,79 @@
-import { computed, ref, useAttrs } from "vue"
+import { computed, useAttrs } from "vue"
 import Uniques from "@/helpers/Uniques"
 
+export interface Input {
+  modelValue?: any
+  error?: string
+  label?: string
+  help?: string
+  placeholder?: string
+}
+
+export interface InputOption {
+  disabled?: boolean
+  help?: string
+  label: string
+  sublabel?: string
+  value: string | number
+}
+
+export interface BooleanInput extends Input {
+  modelValue?: boolean | null
```

Comment:
This used to bite us all the time - now we have `null` support almost across the board as we continue to work with various `Null$t{ype}`'s